kotlintest-core/build.gradle

```diff
@@ -2,4 +2,6 @@ dependencies {
     compile 'org.slf4j:slf4j-api:1.7.25'
     compile "org.jetbrains.kotlin:kotlin-reflect"
     compile 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.1.0'
+
+    compile project(":kotlintest-assertions")
```

Comment:
Don't include assertions with core, not everyone wants them.

Reply:
I thought that everyone always had all of the code anyways, so this wouldn't make a difference. Assertions and Core don't share anything, then? Should I duplicate the Failures class? This doesn't look very good

Reply:
They typically do, but some people only use assertions, and some people only use the framework.

I suppose it's not too bad for core to use assertions actually, but definitely assertions should stand alone.

Perhaps core should be renamed to framework or something.

Reply:
Should we keep this import, then?
Or should I find another way?

Reply:
What do you need from assertions for it to work ?

Reply:
Just using Failures for now... It may be overkill to compile project to this one, but it's also not good to copy Failures or any implementation from there to over here, thus duplicating code

Reply:
Let's (for now) copy failures (at least just the bits you need).
Then for 3.3 we'll make core depend on assertions and maybe rename core to -test-framework or something.

Reply:
Ok, I did it with the latest force push.